Header header 
posePoint[] object_pos